CalorieTracker/CalorieTracker/Extensions/DateFormatter.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DateFormatter.swift
+//  CalorieTracker
+//
+//  Created by Craig Swanson on 2/23/20.
+//  Copyright © 2020 Craig Swanson. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
```

Comment:
Good job practicing good coding habits. The naming convention of the file is clear, and the extension does one thing.